src/css/main.less

```diff
@@ -209,13 +228,57 @@ a,
   padding: 0 0 5px 5px;
 }
 
-/* Application ID cell */
-.table > tbody > tr > td.appid {
-  display: flex;
+/* App list columns */
+.table > colgroup > col {
+
+  &.appid {
+    width: auto;
+  }
+
+  &.ram {
+    width: 96px;
+  }
+
+  &.cpu {
+    width: 80px;
+  }
 
-  .labels {
-    .label {
-      margin-left: @horizontal-spacing-unit;
+  &.status {
+    width: 216px;
+  }
+
+  &.tasks {
+    width: 48px;
+  }
+
+  &.health {
+    width: 6%;
+  }
+
+  &.actions {
+    width: 64px;
+  }
```

Comment:
1. I mixed ``%`` and ``px`` values as it makes no sense to scale all columns, but a few (eg." appid" and "health" column) reveal greater detail and more information if they adapt to different widths.
2. You're right, we should definitely add a "apps" identifier... 
3. I already had in mind to use multiples of our base-horizontal-spacing uint, thus it should be easy to refactor the respective lines.